### PR TITLE
Add version to templates

### PR DESCRIPTION
**Pull request by Amitbergman**
Fixes #

## Proposed Changes

  -
  -
  -

### CODE REVIEW

File: Detections/MultipleDataSources/Solorigate-VM-Network.yaml [hunk 1]
```diff
@@ -75,4 +75,5 @@ entityMappings:
         columnName: DirectoryName
       - identifier: Name
         columnName: Filename
-        
\ No newline at end of file
+
```
Comment (ehudklein) [inline Detections/MultipleDataSources/Solorigate-VM-Network.yaml:78]: should you get rid of the space?

File: .script/tests/detectionTemplateSchemaValidation/Models/ScheduledTemplateInternalModel.cs [hunk 2]
```diff
@@ -23,6 +23,9 @@ public class ScheduledTemplateInternalModel : AnalyticsTemplateInternalModelBase
         [StringLength(10000, MinimumLength = 1)]
         public string Query { get; set; }
 
+        [JsonProperty("version", Required = Required.Always)]
+        public string Version { get; set; }
```
[the remark this answers is not in the document]
Comment (Amitbergman) [reply .script/tests/detectionTemplateSchemaValidation/Models/ScheduledTemplateInternalModel.cs:27]: it is not a guid, it is a version. like "1.0.0"

File: Detections/MultipleDataSources/Solorigate-VM-Network.yaml (hunk 1 above)
Comment (Amitbergman) [reply Detections/MultipleDataSources/Solorigate-VM-Network.yaml:78]: not in this PR

Comment (Amitbergman) [reply Detections/MultipleDataSources/Solorigate-VM-Network.yaml]: Actually why not? Resolved.

**Pull request merged**